### PR TITLE
feat: add support for column widths within tables

### PR DESCRIPTION
**Pull request by nicolasiscoding**

### CODE REVIEW

**Comment by nicolasiscoding** [comment]
Note, this is a fork off of #163 and would recommend rebasing it with whatever changes are made in there

**Comment by nicolasiscoding** [comment]
And it may be easier to see the diffs here: https://github.com/nicolasiscoding/html-to-docx/pull/2

**Comment by privateOmega** [comment]
> Note, this is a fork off of #163 and would recommend rebasing it with whatever changes are made in there

I dont think I can do it, since I wont have write privileges to your fork, please sync your fork against this repo and rebase this branch with develop, since that branch was already merged, the refactoring changes would disappear from this branch.

**Comment by nicolasiscoding** [comment]
> > Note, this is a fork off of #163 and would recommend rebasing it with whatever changes are made in there
> 
> I dont think I can do it, since I wont have write privileges to your fork, please sync your fork against this repo and rebase this branch with develop, since that branch was already merged, the refactoring changes would disappear from this branch.
> 
> While you are at it, please remove the debugging changes as well from here.

@privateOmega Done

File: example/example-node.js [hunk 1]
```diff
@@ -132,6 +132,76 @@ const htmlString =  `<!DOCTYPE html>
                 </td>
             </tr>
         </table>
+
+        <div class="page-break" style="page-break-after: always"></div>
```
Comment (privateOmega) [inline example/example-node.js:136]: add this in examples/example.js and examples/react-example/src/App.js

File: src/helpers/xml-builder.js [hunk 2]
```diff
@@ -251,6 +251,19 @@ const fixupRowHeight = (rowHeightString) => {
   }
 };
 
+// eslint-disable-next-line consistent-return
+const fixupColumnWidth = (columnWidthString) => {
+  if (pointRegex.test(columnWidthString)) {
+    const matchedParts = columnWidthString.match(pointRegex);
+    // convert point to half point
```
Comment (privateOmega) [inline src/helpers/xml-builder.js:258]: remove this

File: example/example-node.js (hunk 1 above)
Comment (nicolasiscoding) [reply example/example-node.js:136]: done

**Review by privateOmega — APPROVED** [review]
LGTM

**Pull request merged**